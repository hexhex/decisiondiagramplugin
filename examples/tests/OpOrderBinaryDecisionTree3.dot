digraph {
     c_1 -> leaf1_2 [label="else"];
     c_2 -> leaf1 [label="else"];
     b_1 -> c_1 [label="else"];
     c_1_1 -> leaf2_2 [label="else"];
     a -> b_1 [label="else"];
     b -> c_2 [label="else"];
     c -> leaf2 [label="else"];
     c_1 -> leaf2_1 [label="c<x"];
     c_2 -> leaf1_1_1 [label="c<x"];
     b_1 -> c_1_1 [label="b<x"];
     c_1_1 -> leaf2_1_1 [label="c<x"];
     a -> b [label="a<x"];
     b -> c [label="b<x"];
     c -> leaf1_1 [label="c<x"];
     leaf1_2 [label="leaf1 [class1]"];
     leaf2_2 [label="leaf2 [class2]"];
     leaf1_1_1 [label="leaf1 [class1]"];
     leaf2_1_1 [label="leaf2 [class2]"];
     leaf1_1 [label="leaf1 [class1]"];
     leaf2_1 [label="leaf2 [class2]"];
     leaf2 [label="leaf2 [class2]"];
     leaf1 [label="leaf1 [class1]"];
     c_1 [label="c"];
     c_2 [label="c"];
     b_1 [label="b"];
     c_1_1 [label="c"];
     a [label="a"];
     b [label="b"];
     c [label="c"];
}
