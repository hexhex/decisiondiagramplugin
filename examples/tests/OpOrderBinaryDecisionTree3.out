{elseedge(b_1,c_1), elseedge(c_2,leaf2), elseedge(c_1,leaf1_2), elseedge(c_1_1,leaf2_2), elseedge(a,b_1), elseedge(b,c), elseedge(c,leaf1), conditionaledge(b_1,c_1_1,b,"<",x), conditionaledge(c_2,leaf1_1_1,c,"<",x), conditionaledge(c_1,leaf2_1,c,"<",x), conditionaledge(c_1_1,leaf2_1_1,c,"<",x), conditionaledge(a,b,a,"<",x), conditionaledge(b,c_2,b,"<",x), conditionaledge(c,leaf1_1,c,"<",x), leafnode(leaf2_2,class2), leafnode(leaf1_2,class1), leafnode(leaf1_1_1,class1), leafnode(leaf2_1_1,class2), leafnode(leaf1_1,class1), leafnode(leaf2_1,class2), leafnode(leaf2,class2), leafnode(leaf1,class1), innernode(b_1), innernode(c_2), innernode(c_1), innernode(c_1_1), innernode(a), innernode(b), innernode(c), root(c)}
